### ISSUE TITLE
Get() hangs when there is more than one job in the queue

### ISSUE THREAD

**Issue by w1am** [peter-wangxu/persist-queue#189]
```
from persistqueue import SQLiteQueue

q = SQLiteQueue(path="/tmp/queue", auto_commit=True)

q.put(item={ "pqid": 1, "name": "william" })
q.put(item={ "pqid": 2, "name": "joe" })

x = q.get(id={ "pqid": 1, "name": "william" })
pprint(x)
```

I tried setting `timeout=0` but it raises Empty error.

**Comment by doliynyk-nference** [comment]
I don't think you cannot set a custom pqid when you create your item. Take a look at q.queue() to see how your data is being stored:
`[{'id': 1, 'data': {'pqid': 100, 'name': 'william'}, 'timestamp': 1669919848.33653}]`
So you are storing a dictionary which contains a pqid entry inside it but the actual id under which your dictionary gets stored under is 1. Thus `x = q.get(id={ "pqid": 100, "name": "william" })` will block indefinitely or raise and Empty error as there is no element stored that has an id of `{ "pqid": 100, "name": "william" }` like you are asking for. On the other hand `q.get(id=1)` will work and return `{'pqid': 100, 'name': 'william'}`.
